keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ActiveServerGroup.kt

```diff
@@ -186,7 +186,7 @@ data class ActiveServerGroupImage(
   ) : this(
     imageId = imageId,
     appVersion = tags.getTag("appversion")?.substringBefore("/"),
-    baseImageVersion = tags.getTag("base_ami_version"),
+    baseImageName = description?.let{ Regex("""ancestor_name=(.+?),""").find(it)?.groupValues?.get(1) },
```

Comment:
Sadly on the active server group we need to scrape the base AMI name out of the image description as there's no convenient tag available

Reply:
yaaaaaay images

Reply:
I suspect the ordering of `description` is consistent _but_ `ancestor_name=([\w-]+)` might also work for the regex should there actually not be a trailing `,`?

Reply:
Hmm, that's a good point. Probably safer.

Reply:
Fixed